src/NServiceBus.AcceptanceTesting/Support/EndpointRunner.cs

```diff
@@ -173,13 +173,13 @@ void RegisterInheritanceHierarchyOfContextInSettings(ScenarioContext context)
 
         async Task Cleanup()
         {
-            dynamic transportCleaner;
+            IConfigureTestExecution transportCleaner;
             if (busConfiguration.GetSettings().TryGet("CleanupTransport", out transportCleaner))
             {
                 await transportCleaner.Cleanup();
             }
 
-            dynamic persistenceCleaner;
+            IConfigureTestExecution persistenceCleaner;
             if (busConfiguration.GetSettings().TryGet("CleanupPersistence", out persistenceCleaner))
```

Comment:
Do we need to separate them like this or could we just have a 
```
List<IConfigureTestExecution> cleaners;
busConfiguration.GetSettings().TryGet(out cleaners)
```
instead?

That way we can just add more in the future without touching this code? (or does the order matter in some way)